doc/tutorials/run_kata_containers.rst

```diff
@@ -0,0 +1,329 @@
+.. _run-kata-containers:
+
+How to run Kata containers on Service VM
+########################################
+
+This tutorial describes how to install, configure and run Kata containers on
+ACRN Service VM.
+
+Validated Version
+*****************
+
+- Clear Linux version: **31670**
+- Docker Server Version: **19.03.2**
+- Kata Environment tag (Runtime/Proxy/Shim/Agent/Netmon): **1.10.0-alpha0**
+
+Introduction
+************
+
+`Kata Containers <https://katacontainers.io/>`_ is an open source project
+and community working to build a standard implementation of lightweight
+Virtual Machines (VMs) that feel and perform like containers, but provide
+the workload isolation and security advantages of VMs.
+
+Kata Code Repositories
+======================
+
+Kata containers code repositories: https://github.com/kata-containers/kata-containers
+
+.. list-table:: Kata Containers-developed components
+   :header-rows: 1
+   :widths: auto
+
+   * - Component name
+     - Description
+   * - **Agent**
+     - The `kata-agent <https://github.com/kata-containers/agent>`_ runs inside the virtual
+       machine and sets up the container environment.
+   * - **Proxy**
+     - The `kata-proxy <https://github.com/kata-containers/proxy>`_ is a process that runs
+       on the host and co-ordinates access to the agent running inside the virtual machine.
+   * - **Runtime**
+     - The `kata-runtime <https://github.com/kata-containers/runtime>`_ is usually invoked
+       by a container manager and provides high-level verbs to manage containers.
+   * - **Shim**
+     - The `kata-shim <https://github.com/kata-containers/shim>`_ is a process that runs on
+       the host. It acts as though it is the workload (which actually runs inside the virtual machine).
+       This shim is required to be compliant with the expectations of the
+       `OCI runtime specification <https://github.com/opencontainers/runtime-spec>`_.
+   * - **Packaging**
+     - We use the `packaging <https://github.com/kata-containers/packaging>`_ repository to
+       create packages for the `system components
+       <https://github.com/kata-containers/kata-containers#kata-containers-developed-components>`_
+       including `rootfs <https://github.com/kata-containers/kata-containers#os-builder>`_ and
+       `kernel <https://github.com/kata-containers/kata-containers#kernel>`_ images.
+   * - **OS builder**
+     - The `osbuilder <https://github.com/kata-containers/osbuilder>`_ tool can create a rootfs
+       and a "mini O/S" image. This image is used by the hypervisor to setup the environment
+       before switching to the workload.
+
+Pre-requisites
+**************
```

Comment:
I also would like to see a paragraph or two that talk about some of the current limitations, namely:
1. This is only available for the SDC scenario
2. You must have 4 CPUs cores (hyperthread OK) available